docs/content/manual/v1.6/manual.yml

```diff
@@ -1374,7 +1374,7 @@ sections:
         body: |
 
           The filter `contains(b)` will produce true if b is
-          completely contained within the input. A string B is
+          completely contained within the input. String B is
           contained in a string A if B is a substring of A. An array B
```

Comment:
completely contained within the input. String B is
          contained in string A if B is a substring of A. An array B

Reply:
Sorry I do not know what is happening to me but I neither see the option on github to merge your change nor do I actually see what you changed.